docs/install.md

```diff
@@ -143,6 +157,14 @@ For example, you can do the following to have Prettier run before each commit:
    node --eval "fs.writeFileSync('.husky/pre-commit','pnpm exec lint-staged\n')"
    ```
 
+  <!--bun-->
+
+   ```bash
+   bun add --dev husky lint-staged
+   bunx husky init
```

Comment:
As I tested on Windows this command fails 

```js
>bunx husky init
<CWD>/node_modules/husky/bin.mjs:18
        w('.husky/pre-commit', p.env.npm_config_user_agent.split('/')[0] + ' test\n')
                                                           ^

TypeError: Cannot read properties of undefined (reading 'split')
    at <CWD>/node_modules/husky/bin.mjs:18:53
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)

Node.js v20.10.0
```

Node.js issue?

Reply:
I've also managed to reproduce this issue on the latest version of Node.js, so I'm also wondering about it.